airbyte-local.sh

```diff
@@ -919,9 +932,12 @@ main() {
     validateRequirements
     setTheme
     validateInput
-    tempPrefix="tmp-$(jq -r -n "now")"
+    timestamp=$(jq -r -n "now")
+    tempPrefix="tmp-$timestamp"
     tempPath="$(pwd)/$tempPrefix"
     tempdir=$(mkdir -p $tempPath && echo $tempPath)
+    src_container_name="airbyte-local-src-$timestamp"
```

Comment:
how does the name would look like?

Reply:
`airbyte-local-src-1718314512.479648`

Reply:
does docker host allow that? i.e character set wise, length wise.

Reply:
yes i took those restrictions into consideration

Reply:
🏆